test/integration/install_scripts_int_test.go

```diff
@@ -126,6 +127,33 @@ func (suite *InstallScriptsIntegrationTestSuite) TestInstall() {
 	}
 }
 
+func (suite *InstallScriptsIntegrationTestSuite) TestInstall_NonEmptyTarget() {
+	suite.OnlyRunForTags(tagsuite.InstallScripts)
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	script := scriptPath(suite.T(), ts.Dirs.Work)
+	argsPlain := []string{script, "-t", ts.Dirs.Work}
+	argsPlain = append(argsPlain, "-b", constants.BranchName)
+	argsWithActive := append(argsPlain, "-f")
+	var cp *termtest.ConsoleProcess
+	if runtime.GOOS != "windows" {
+		cp = ts.SpawnCmdWithOpts(
+			"bash", e2e.WithArgs(argsWithActive...),
+			e2e.AppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"),
+		)
+	} else {
+		cp = ts.SpawnCmdWithOpts("powershell.exe", e2e.WithArgs(argsWithActive...),
+			e2e.AppendEnv("SHELL="),
+			e2e.AppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"),
+		)
+	}
+	cp.ExpectLongString("Installation path must be an empty directory")
+	if runtime.GOOS != "windows" {
+		cp.ExpectExitCode(1)
```

Comment:
Should file a bug; Windows should still exit non-zero.

Reply:
Created here: https://activestatef.atlassian.net/browse/DX-826